### PR TITLE
Slideshow block: Remove "fade" transition slide background

### PR DESCRIPTION
**Pull request by ivan-ottinger**
#### Changes proposed in this Pull Request:
This PR addresses the issue https://github.com/Automattic/jetpack/issues/12962.

After a discussion on the [issue thread](https://github.com/Automattic/jetpack/issues/12962#issuecomment-916208478), we decided to propose a CSS change that removes the default background color from slides and makes sure that only the active slide is being displayed.

#### Does this pull request change what data or activity we track or use?
No, it does not.

#### Testing instructions:
1. Create a Slideshow Block
2. Change the transition effect to `Fade` and publish the post/page
3. View the post/page. There should be no background around slides, the page/post background color should be used.

Before:
Markup on 2021-09-10 at 10:06:16

After:
Markup on 2021-09-10 at 10:05:05

Fixes https://github.com/Automattic/jetpack/issues/12962.

### CODE REVIEW

**Review by yansern — DISMISSED** [review]
Works well!

https://user-images.githubusercontent.com/1287077/132839903-5d77ed64-2d68-4a46-a3ee-5f94402f33ff.mp4

File: projects/plugins/jetpack/extensions/blocks/slideshow/style.scss [hunk 1]
```diff
@@ -64,9 +64,10 @@
 		}
 	}
 
-	.swiper-container-fade .wp-block-jetpack-slideshow_slide {
-		background: var( --color-neutral-0 );
-	}
+	// When the slideshow transition is set to "fade", hide all slides that are not active.
+	.swiper-container-fade .wp-block-jetpack-slideshow_slide:not(.swiper-slide-active) {
+        opacity: 0 !important;
+    }
```
Comment (jeherve) [inline projects/plugins/jetpack/extensions/blocks/slideshow/style.scss:70]:
```suggestion
		opacity: 0 !important;
	}
```

**Review by jeherve — CHANGES_REQUESTED** [review]

**Review by jeherve — APPROVED** [review]
This works out just fine, without showing the other slides. 🚢

**Comment by ivan-ottinger** [comment]
Successfully deployed to WordPress.com. Changeset ID: r231567-wpcom

**Pull request merged**